### ISSUE TITLE
Testing Report: New deb packages on Xubuntu 22.04/23.10 stuck at "retrieving current status"

### ISSUE THREAD

**Issue by makeasnek** [BOINC/boinc#5431]
I am testing the new deb packages on various linux variants to see if they work.

Test environment:
Xubuntu 22.04.3 Desktop x64 running on live usb in Virtualbox. I am using xubuntu for performance reasons but I imagine this should not make a big difference. This is testing with the jammy package nightly.

Test notes:
- boinc-client systemd service not started automatically after install 
- menu icon for BOINC manager is missing. BOINC Manager shows in the app menu, but the icon is the 'placeholder' icon
- Opening BOINC manager from application menu works but is stuck at "retrieving current status" because boinc-client is not running. It stays there forever. Sometimes it gives error `gui_rpc_auth.cfg not found`.
- Starting boinc-client with `systemctl start boinc-client` gives error (screenshot attached). 
- I can run `boinc` from command line successfully and it starts. 
- Starting `boincmgr` from terminal works! I was able to successfully attach to a project
- Edit: Behavior on 23.10 and 20.04 (using focal release) is identical.
![2204](https://github.com/BOINC/boinc/assets/90811423/5b1ee8f3-96b6-4f8b-9ba6-c665a148cf5e)

I can confirm at least in 20.04 that /usr/local/bin/boinc does not exist

**Comment by AenBleidd** [comment]
@makeasnek, thank you for the report.
I'll take care about it

**Comment by RichardHaselgrove** [comment]
> * Starting boinc-client with `systemctl start boinc-client` gives error (screenshot attached).

Your screenshot shows `boinc-client.service: disabled` - have you tested `systemctl enable boinc-client` before `start`?

**Comment by makeasnek** [comment]
> > * Starting boinc-client with `systemctl start boinc-client` gives error (screenshot attached).
> 
> Your screenshot shows `boinc-client.service: disabled` - have you tested `systemctl enable boinc-client` before `start`?

Yes just tried now, same result

**Comment by gj82854** [comment]
I think some of these issues are duplicates of #5335 (which would possibly cover the gui_rpc_auth issue), #5325 (which would cover the systemctl issue), and possibly #5324 which might contribute to client start issues. These were reported against the 7.24.x release

**Comment by Toby-Broom** [comment]
I did a clean install of 23.10 and it was sort of similar with 7.20.5 thats deployed with that version, I built 7.24.1 from source code and the issue seems fixed, so I would say that gj82854 comments could be correct in there is some rpc issue in the older version

**Comment by gj82854** [comment]
I took the .rpm from the alpha repo and broke it apart into it's individual files. The files that are distributed with the packages have the errors as indicated. I don't think the problem is with the client code itself but with the packaging process used to build the .deb and .rpm files. I am currently running the 7.24.1 version of the client from the repo but I manually corrected all the issues that prevented the client from starting. Once I got it to start through systemctl, it has been running quite nicely on Fedora 38.
